manifests/crc/default-db-included-installation/openshift/kruize-crc-openshift.yaml

```diff
@@ -75,10 +75,15 @@ data:
         "adminUsername": "admin",
         "hostname": "kruize-db-service",
```

Comment:
I guess this PR is limited to datasource auth , But not for DB ?

Reply:
Reverted